modules/visual_script/visual_script.cpp

```diff
@@ -1430,20 +1280,20 @@ bool VisualScriptInstance::get(const StringName &p_name, Variant &r_ret) const {
 }
 
 void VisualScriptInstance::get_property_list(List<PropertyInfo> *p_properties) const {
-	for (const Map<StringName, VisualScript::Variable>::Element *E = script->variables.front(); E; E = E->next()) {
-		if (!E->get()._export) {
+	List<StringName> vars;
+	script->variables.get_key_list(&vars);
+	for (const List<StringName>::Element *E = vars.front(); E; E = E->next()) {
+		if (!script->variables[E->get()]._export)
 			continue;
```

Comment:
```suggestion
		if (!script->variables[E->get()]._export) {
			continue;
		}
```
There might be more, make sure that all single-line blocks use brackets as required by our style guide now.

Reply:
Ahh thanks for it, almost forgot, will review and fix up all of them.

Reply:
Ping!